tools/disasm/disasm.py

```diff
@@ -26,10 +33,14 @@
     required=False,
     help="Optional list of files to diassemble separated by a space. This is a whitelist, all files will be skipped besides the ones listed here if used.",
 )
+parser.add_argument("-Mreg-names", choices=fpr_name_options.keys(), help="How to name registers in the output")
```

Comment:
This is a long argument name, so it should have 2 `-`, doesn't it?
```suggestion
parser.add_argument("--Mreg-names", choices=fpr_name_options.keys(), help="How to name registers in the output")
```

Reply:
It mirrors the flag from objdump, that's why it's not POSIX.

Reply:
That objdump flag is POSIX complaint. The flag itself is `-M`. From its man page:
```
       -M options
       --disassembler-options=options
           Pass target specific information to the disassembler.  Only supported on some targets.  If it is
           necessary to specify more than one disassembler option then multiple -M options can be used or can be
           placed together into a comma separated list.

```

Reply:
Speaking of argument names, the `files` argument above should probably also be adjusted (though might be out of scope of this PR). In particular I would suggest the `-f/--full` flag be changed to `-a/-all` and the `-files/--files` flag be `-f/--files`.

Reply:
Oh goodness, yes the files one should change.